contrib/opencensus-ext-azure/opencensus/ext/azure/metrics_exporter/heartbeat_metrics/heartbeat.py

```diff
@@ -1,132 +0,0 @@
-# Copyright 2019, OpenCensus Authors
```

Comment:
Why removing heartbeat?, I believe is still relevant and is not being replaced by Statsbeat

Reply:
Heartbeat was never fully implemented/committed for OpenCensus Python. There are no current plans to implement them either.